curvefs/src/client/client_operator.cpp

```diff
@@ -418,8 +419,6 @@ CURVEFS_ERROR RenameOperator::UpdateInodeCtime() {
 }
 
 void RenameOperator::UpdateCache() {
-    dentryManager_->DeleteCache(parentId_, name_);
```

Comment:
> Why is this step unnecessary?

yep, because:
* it works well if you only wants to show files under directory, like `ls -l`.
* we control all cache consistent by cache timeout, you can adjust cache timeout for some specified cases which sensitive to the `attribute`.